test/built-ins/Promise/any/reject-immed.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2019 Sergey Rubanov. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Promise.any([]) rejects immediately
+esid: sec-promise.any
+flags: [async]
+includes: [promiseHelper.js]
+features: [Promise.any]
+---*/
+
+Promise.any([])
+  .then(function() {
+    $DONE('The promise should not be fulfilled.');
+  }, $DONE);
```

Comment:
If Promise.any([]) should reject immediately, this test will fail, because $DONE will be called with an error instead of `undefined`. You might want to write: `() => $DONE()`, but furthermore, you might want to include an assertion about the error before `$DONE` is called. It would also be nice to include the spec text where this error is being thrown from.